public/persona.js

```diff
@@ -472,4 +475,24 @@ $(document).ready(function () {
 			}
 		});
 	}
+
+	function tweakBsCollapse() {
+		$('[data-toggle="collapse"]').off('click', _tweakBsCollapse).on('click', _tweakBsCollapse);
+	}
+
+	function _tweakBsCollapse({ currentTarget: el }) {
+		const isVisible = $(el.getAttribute('data-target')).is(':visible');
+		el.classList.toggle('caret-up', !isVisible);
+		const textVariant = el.getAttribute('data-text-variant');
+		if (!textVariant) {
+			return;
+		}
+		el.setAttribute('data-text-variant', el.textContent);
+		for (const child of el.childNodes) {
+			if (child.textContent === el.textContent) {
+				child.textContent = textVariant;
+				break;
+			}
+		}
+	}
```

Comment:
Perhaps we can come up with a more elegant solution than this... not sure what this does?